includes/abstracts/abstract-wc-payment-token.php

```diff
@@ -16,7 +16,7 @@
  * @category	Abstract Class
  * @author		WooThemes
  */
- abstract class WC_Payment_Token implements WC_Data {
+ abstract class WC_Payment_Token {
```

Comment:
Yeah, this won't break anything :) Maybe just need to tweak shipping zones which implemented WC_Data too.